Styles/var.css

```diff
@@ -0,0 +1,8 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+*{
+    font-size: 1rem;
+    line-height: 1.3rem;
+}
```

Comment:
Used * selector. These styles will be applied to all selectors. Are you sure about this?

Reply:
Yes This is the basic size of all my text nd if I need any specific size I have changed accordingly

Reply:
I have approved the PR. You can merge if you're sure about this.